modules/nw-ingress-gateway-api-enable.adoc

```diff
@@ -0,0 +1,65 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/ingress-gateway-api.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-gateway-api-enable_{context}"]
+= Enabling Gateway API for the Ingress Operator
+
+When you enable Gateway API, it installs the subscription, the Istio operator, the custom resource defintions (CRDs), and creates the Istio resources. The following procedure describes how to enable Gateway API.
+
+.Procedure
+
+. Create a `GatewayClass` object with the following manifest:
++
+[source,yaml]
+----
+apiVersion: gateway.networking.k8s.io/v1
+kind: GatewayClass
+metadata:
+  name: openshift-default
+spec:
+  controllerName: openshift.io/gateway-controller/v1 <1>
+----
+<1> The controller name must be `v1` for the Ingress Operator to manage it. Setting this field to anything else will be ignored by the Ingress Operator, because it is assumed to be a different implementation.
```

Comment:
```suggestion
<1> The controller name must be exactly as shown for the Ingress Operator to manage it. Setting this field to anything means the GatewayClass and all of its associated Gateways, GRPCRoutes, and HTTPRoutes will be ignored by the Ingress Operator.  The controllerName is tied to the implementation and `openshift.io/gateway-controller/v1` is the only controllerName that can be used with the OpenShift implementation.
```

Reply:
Actually, this little note should be highlighted as an "Important" note.  We don't want anyone to miss this.

Reply:
I think it's important to keep the context, so the callout will stand, but we can add an `IMPORTANT` admonition in a way that is chained to the callout for extra visibility